Documentation/Usage.md

```diff
@@ -393,6 +393,31 @@ import Foundation
 #endif
 ```
 
+## Using Mirrored packages
+
+To add mirror for a given package:
+```sh
+$ swift package config set-mirror --package-url https://github.com/apple/swift-nio.git --mirror-url ../LocalDeps/nio
```

Comment:
One of the primary use-case of this feature is mirroring the repository on some other server. I think it'll be better to use mirror-url which is an actual url. Maybe something like this:

```sh
swift package config set-mirror --package-url https://github.com/apple/swift-nio.git --mirror-url https://github.com/<your-org>/swift-nio.git
```